pom.xml

```diff
@@ -32,32 +31,36 @@
         </license>
     </licenses>
 
-    <developers>
-        <developer>
-            <id>michelzanini</id>
-            <name>Michel Zanini</name>
-        </developer>
-    </developers>
-
     <scm>
-        <connection>scm:git:git://github.com/jenkinsci/bitbucket-filter-project-trait-plugin.git</connection>
+        <connection>scm:git:https://github.com/jenkinsci/bitbucket-filter-project-trait-plugin.git</connection>
         <developerConnection>scm:git:git@github.com:jenkinsci/bitbucket-filter-project-trait-plugin.git</developerConnection>
         <url>https://github.com/jenkinsci/bitbucket-filter-project-trait-plugin</url>
         <tag>${scmTag}</tag>
     </scm>
 
+    <dependencyManagement>
+      <dependencies>
+        <dependency>
+          <groupId>io.jenkins.tools.bom</groupId>
+          <artifactId>bom-${jenkins.baseline}.x</artifactId>
+          <version>4890.vfca_82c6741a_d</version>
+          <type>pom</type>
+          <scope>import</scope>
+        </dependency>
+      </dependencies>
+    </dependencyManagement>
+
     <dependencies>
 
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>cloudbees-bitbucket-branch-source</artifactId>
-            <version>${bitbucket-branch-source.version}</version>
         </dependency>
 
         <dependency>
             <groupId>com.jcraft</groupId>
```

Comment:
The dependency to jcraft jsch is not needed at all and should be removed. Project builds fine without it.

Reply:
I have removed it now